vmc/constants.go

```diff
@@ -14,8 +14,9 @@ const (
 	SksNSXTManager string = "/sks-nsxt-manager"
 
 	// ESX Host instance types supported for SDDC creation.
-	HostInstancetypeI3 string = "I3_METAL"
-	HostInstancetypeR5 string = "R5_METAL"
+	HostInstancetypeI3   string = "I3_METAL"
```

Comment:
what happens if we use "i3.metal"?

Reply:
Ideally this matches the VMC API which uses all lower case. Right now, it'll fail the operation and state that its looking for upper case value